Web/scripts/fluid_properties.IF97Conformance.py

```
@@ -217,6 +248,71 @@ def _read_props_fast(state, h, p, prop_int_keys):
+    residual against the IF97 forward equations, not the reference's
+    own backward floor.
+
+    State pinning: hold the IF97 phase across the Newton iteration so
+    HEOS / IF97 doesn't flip basins mid-iteration on probes near the
+    saturation curve.
+
+    Best-effort: returns silently after max_iter (5 is sufficient for
+    Newton on a smooth h(T, p) — typical convergence is 2-3 iters)."""
+    try:
+        state.update(CP.HmassP_INPUTS, h_target, p)
+    except Exception:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)